dts/bindings/pinctrl/realtek,rts5912-pinctrl.yaml

```
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+description: |
+     This binding gives a base representation of the pins configuration
+
+compatible: "realtek,rts5912-pinctrl"
+
+include: [base.yaml, pinctrl-device.yaml, pincfg-node.yaml]
+
+properties:
+  reg:
+    required: true
+
+child-binding:
+  description: |
+    This binding gives a base representation of the pins configuration
+
+  include:
+    - name: pincfg-node.yaml
+      property-allowlist:
+        - bias-pull-down
+        - bias-pull-up
+        - drive-push-pull
+        - drive-open-drain
+        - input-enable
+        - output-enable
+        - output-high
+        - output-low
+        - input-schmitt-enable
+
+  properties:
+    pinmux:
+      type: int
+      required: true
+      description: Pinmux selection
+    drive-strength:
+      type: string
+      enum:
+        - "low"
+        - "high"
+      description: |
+        0 low drive strength
+        1 high drive strength
```

The `drvie-strength` property type is string, but description field uses integers.  

```suggestion
        "low" - 4mA/ 8mA drive strength
        "high" - 8mA/12mA drive strength
```

---

Because of the conflicts, we add this changes to this PR, thanks.